### DISCUSSION TITLE
nebula formula is missing `-service` option

### DISCUSSION THREAD

**Discussion by stangri** [Homebrew/discussions#2731] [Everyday usage] [answered]
If I download the `nebula-darwin.zip` from [releases](https://github.com/slackhq/nebula/releases) the `nebula` binary has a `-service` option allowing for set and forget running of nebula.

However if I install `nebula` from brew, that option is missing:

```sh
$ which -a nebula
/usr/local/sbin/nebula
/opt/homebrew/bin/nebula
$ /usr/local/sbin/nebula
-config flag must be set
Usage of /usr/local/sbin/nebula:
  -config string
    	Path to either a file or directory to load configuration from
  -help
    	Print command line usage
  -service string
    	Control the system service.
  -test
    	Test the config and print the end result. Non zero exit indicates a faulty config
  -version
    	Print version
$ /opt/homebrew/bin/nebula
-config flag must be set
Usage of /opt/homebrew/bin/nebula:
  -config string
    	Path to either a file or directory to load configuration from
  -help
    	Print command line usage
  -test
    	Test the config and print the end result. Non zero exit indicates a faulty config
  -version
    	Print version
$
``` 

I'm not familiar with underpinnings of brew enough to figure out to which repo and who has committed support for nebula, so if anyone can help by fixing the `nebula` formula I'd appreciate it.

**Comment by alebcay** [comment, marked as answer]
Thanks for bringing this up, proposed a fix in https://github.com/Homebrew/homebrew-core/pull/92667.

**Reply by stangri** [reply]
Thanks, I'm not savvy enough in brew to test before it's merged, but I can after.

**Reply by alebcay** [reply]
The change has been merged, you should be able to pick it up with `brew update`/`brew upgrade`. Feel free to mark as resolved if it works now or comment further if something's still wrong!
